frontend/src/App.test.tsx

```diff
@@ -62,9 +70,9 @@ test("data is fetched again after refresh interval", async () => {
 
   await waitFor(() => expect(fetch).toHaveBeenCalledTimes(1));
 
-  screen
-    .getAllByText("31 °C", { exact: false })
-    .forEach((temp) => expect(temp).toBeInTheDocument());
+  (await screen.findAllByText("31 °C", { exact: false })).forEach((temp) =>
```

Comment:
NIT: Might be worth having a more descriptive name here than `temp`

Reply:
I actually meant `temp` for `temperature` not `temporary` lol, but it does slightly jump out. Renamed to `gpu_temp`.